CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ To set up a local development environment:
    ```
 2. Install dependencies
    ```bash
-   pip install -r requirements.txt
+   pip install -r requirements_dev.txt
```

Comment:
I haven't used `pip install -e .'[dev]'` in `test-current-version.yml` because from what I understand, pytest is not necessary in those tests and I wanted to stay as close as possible to "real" conditions, where the client installs `tofupilot` without the dev dependencies.
However it only works because we wanted to disable the pytest plugin for now. If we want to include it again, we'll have to change the dependencies VS dev dependencies.